src/test/unit_tests/jest_tests/test_nc_account_invalid_mkm_integration.test.js

```diff
@@ -270,8 +267,6 @@ function fail(reason) {
 }
 
 async function setup_nc_system_and_first_account() {
-    await P.all(_.map([CONFIG_SUBDIRS.ACCOUNTS, CONFIG_SUBDIRS.ACCESS_KEYS], async dir =>
-        fs_utils.create_fresh_path(`${config_root}/${dir}`)));
```

Comment:
What is replacing this?

Reply:
the first time the CLI will be called will create the config directory